<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="buttonsbar"/>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="playerbar">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="playbutton">
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pausebutton">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopbutton">
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="length">
        <property name="text">
         <string>  no song played</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionaddsongs"/>
   <addaction name="actionDeletesongs"/>
   <addaction name="actionFindallsongs"/>
   <addaction name="actionEditmode"/>
   <addaction name="actionShow_hidden_columns"/>
   <addaction name="actionShow_all_entries"/>
  </widget>
  <action name="actionaddsongs">
   <property name="icon">
    <iconset>
     <normaloff>F:/addsongsfix.png</normaloff>F:/addsongsfix.png</iconset>
   </property>
   <property name="text">
    <string>add songs</string>
   </property>
  </action>
  <action name="actionDeletesongs">
   <property name="icon">
    <iconset>
     <normaloff>F:/deletesongs.png</normaloff>F:/deletesongs.png</iconset>
   </property>
   <property name="text">
    <string>delete songs</string>
   </property>
  </action>
  <action name="actionFindallsongs">
   <property name="icon">
    <iconset>
     <normaloff>F:/addsongsiconfix.png</normaloff>F:/addsongsiconfix.png</iconset>
   </property>
   <property name="text">
    <string>find all songs</string>
   </property>
  </action>
  <action name="actionEditmode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Bine/Downloads/editnoback.png</normaloff>../../../Bine/Downloads/editnoback.png</iconset>
   </property>
   <property name="text">
    <string>edit mode</string>
   </property>
  </action>
  <action name="actionShow_hidden_columns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Bine/Downloads/showcollumns.png</normaloff>../../../Bine/Downloads/showcollumns.png</iconset>
   </property>
   <property name="text">
    <string>show hidden columns</string>
   </property>
  </action>
  <action name="actionShow_all_entries">
   <property name="text">
    <string>show all entries</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
